src/prop/theory_proxy.cpp

```diff
@@ -101,7 +101,10 @@ void TheoryProxy::presolve()
   d_stopSearch = false;
 }
 
-void TheoryProxy::postsolve() { d_theoryEngine->postsolve(); }
+void TheoryProxy::postsolve(SatValue result)
```

Comment:
I'm confused: I thought that this change was already made in the previous PR for the partitioning work.